common/src/main/java/com/sequenceiq/cloudbreak/validation/FunctionChecked.java

```diff
@@ -0,0 +1,7 @@
+package com.sequenceiq.cloudbreak.validation;
+
+@FunctionalInterface
+public interface FunctionChecked<T, R, E extends Exception> {
```

Comment:
Why not CheckedFunction? Or ValidatedFunction? Would sound more correct.

Reply:
bad package, I would create in util. I prefer the function first because to find easily by another one

Reply:
But this name does not make sense. You can find it anyway.